waypoint_x: [0,0,0.5,1,1]
waypoint_y: [0,1,1.5,1,0]
waypoint_th: [-1.57079, -0.785398, -1.57079, -0.785398, -1.57079]
freq: 120